wx/py/tests/test_introspect.py

```diff
@@ -522,18 +527,18 @@ def setUp(self):
             'fileno',
             'find',
             'flush',
-            'func_closure',
-            'func_code',
-            'func_defaults',
-            'func_dict',
-            'func_doc',
-            'func_globals',
-            'func_name',
+            # 'func_closure',
+            # 'func_code',
+            # 'func_defaults',
+            # 'func_dict',
+            # 'func_doc',
+            # 'func_globals',
+            # 'func_name',
```

Comment:
Let's remove rather than comment out.

Reply:
Cleaned up the comments and fixed a typo in the first commit message.